packages/salesforcedx-vscode-apex/package.json

```diff
@@ -25,6 +25,7 @@
   ],
   "dependencies": {
     "@salesforce/apex-tmlanguage": "1.4.0",
+    "@salesforce/apex-node": "0.1.9",
```

Comment:
I'd like to wait until the new version of the Apex library is released so we can take advantage of the auth issue fix - so this version number will be updated